uiowa/uiowa
# Minimal HTML default text format

Resolves #2038, relates to #1771 

Sets Minimal HTML over Filtered HTML as the default format for formatted text fields site-wide. This default is overridden for new Text Area blocks, which are set to Filtered HTML.

## Testing

- Sync a site/import config changes.
- Check that no existing content blocks are broken (specifically, blocks which had used Filtered HTML, such as the Hero block on uiowa.edu)--no existing block text format should change.
- Create new blocks. 
  - Text format for all but Text Area blocks should be Minimal HTML. 
  - Blocks such as Hero which allow Filtered should default to Minimal HTML, but allow for changing to Filtered HTML.
  - Change a Hero or other block to use Filtered HTML and save--check that it saves as Filtered HTML.
- Text Area blocks should default to Filtered HTML.
